cmake/kokkos_enable_options.cmake

```diff
@@ -27,7 +27,11 @@ KOKKOS_CFG_DEPENDS(OPTIONS COMPILER_ID)
 KOKKOS_DEPRECATED_LIST(OPTIONS ENABLE)
 
 # Set the Default for Desul Atomics usage.
+IF (KOKKOS_ENABLE_OPENACC)
+set(_DESUL_ATOMICS_DEFAULT OFF)
```

Comment:
We disabled the _DESUL_ATOMICS_DEFAULT since the OpenACC atomic implementations are not yet included in the desul library, assuming that the desul library is an external third-party one. If adding the OpenACC atomic implementation into the desul library is allowed, we can remote the above code.